polaris-core/src/main/java/org/apache/polaris/core/persistence/pagination/Page.java

```diff
@@ -24,6 +24,7 @@
 import java.util.ArrayList;
 import java.util.Iterator;
 import java.util.List;
+import java.util.function.BooleanSupplier;
```

Comment:
Do we really need to import `BooleanSupplier` here as it's not really used ?

Reply:
It's referenced from javadoc :)

Reply:
I believe using imports in javadoc is standard in Polaris... at least my IDE does that automagically :)

Reply:
Yes, I see that, but it looks weird to import just for a javadoc 😄

Reply:
Make the javadoc text more concise. I kind it like it personally :)